frontend/homework/1/style.css

```diff
@@ -0,0 +1,176 @@
+* {
+  margin: 5px;
+}
+
+#form {
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  margin: 0 auto;
+  padding: 20px;
+  border: #000;
+  background: #fff;
+  border-radius: 5px;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+  overflow: hidden;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+}
+
+.header {
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: 20px;
+  background-color: rgb(9, 9, 145);
+  color: white;
+}
+
+input {
+  display: block;
+  width: 100%;
+  padding: 5px;
+  border: 1px solid #969393;
+  border-radius: 5px;
+  box-sizing: border-box;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+}
+
+#border-box {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100px;
+  padding: 5px;
+  border: 1px solid #969393;
+  border-radius: 5px;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 20px auto;
+  text-align: center;
+}
+
+#content-box {
+  box-sizing: content-box;
+  width: 100%;
+  height: 100px;
+  padding: 5px;
+  border: 1px solid #969393;
+  border-radius: 5px;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 20px auto;
+  text-align: center;
+}
+
+#big-box {
+  width: 100%;
+  height: auto;
+  padding: 5px;
+  border: 1px solid #969393;
+  border-radius: 5px;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 20px auto;
+  text-align: center;
+}
+
+#relative {
+  width: auto;
+  height: 100px;
+  padding: 5px;
+  border: 1px solid #000000;
+  border-radius: 5px;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 20px auto;
+  text-align: center;
+}
+
+#fixed {
+  position: -webkit-sticky;
```

Comment:
Since we are using something specific to a browser, try to add similar properties for other browsers as well like `-moz-`(Mozilla), `-ms-`(Edge), `-o-`(Opera)